sdk/search/Microsoft.Azure.Search.Service/src/Generated/Models/VisualFeature.cs

```diff
@@ -28,11 +28,7 @@ public enum VisualFeature
         [EnumMember(Value = "description")]
         Description,
         [EnumMember(Value = "faces")]
-        Faces,
-        [EnumMember(Value = "imageType")]
-        ImageType,
-        [EnumMember(Value = "color")]
-        Color
+        Faces
```

Comment:
We don't generally modify generated code by hand. Can we just remove it from the tests to unblock the management SDK? We weren't planning to ship a new version of the old SDK anyway.

Reply:
Done in commit https://github.com/Azure/azure-sdk-for-net/pull/12375/commits/450059759567fee2ced7d5ad9d846c497b828a34